ISICkonto/AppDelegate.swift

```diff
@@ -7,40 +7,74 @@
+            window.rootViewController = UINavigationController()
+            appFlowController = AppFlowController(with: window.rootViewController as! UINavigationController)
+            appFlowController!.launch()
+            window.makeKeyAndVisible()
+        }
+        
         return true
     }
-
+    
     func applicationWillResignActive(_ application: UIApplication) {
         // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
         // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
     }
-
+    
     func applicationDidEnterBackground(_ application: UIApplication) {
         // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
         // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
     }
-
+    
     func applicationWillEnterForeground(_ application: UIApplication) {
         // Called as part of the transition from the background to the active state; here you can undo many of the changes made on entering the background.
     }
-
+    
     func applicationDidBecomeActive(_ application: UIApplication) {
         // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
     }
-
+    
     func applicationWillTerminate(_ application: UIApplication) {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
     }
-
-
+    
+    func setUpGlobalAppearance() {
+        
+        // for UINavigationBar
+        UINavigationBar.appearance().setBackgroundImage(UIImage(), for: UIBarMetrics.default)
+        UINavigationBar.appearance().shadowImage = UIImage()
+        UINavigationBar.appearance().isTranslucent = true
+        UINavigationBar.appearance().backgroundColor = UIColor.clear
+        UINavigationBar.appearance().tintColor = UIColor.white
+        UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white, NSAttributedString.Key.font: UIFont(name: "Cabin-Regular", size: 20)!]
+        UIBarButtonItem.appearance().setTitleTextAttributes([NSAttributedString.Key.foregroundColor: UIColor.white, NSAttributedString.Key.font: UIFont(name: "Cabin-Medium", size: 17)!], for: [.normal, .highlighted])
+        
+        // for SVProgressHUD
+        SVProgressHUD.setFont(UIFont(name: "Cabin-Regular", size: 15)!)
```

Comment:
pro tip: Když si uděláš vlastní extension nad `UIFont`, ze které ti bude padat custom font (třeba `UIFont.cabinForSize(_:)`), budeš mít načítání custom fontu na jednom místě a současně ten vykřičník taky bude na jednom místě.

Reply:
jasne, to jsem mela v planu udelat, ale nejak jsem na to zapomnela, a aspon by tam nebyl ten String 🙃